### ISSUE TITLE
Run on Colab? or add to Auto1111 repo as extension? 

### ISSUE THREAD

**Issue by Maki9009** [hardmaru/-image-notebook#3]
anyway to run the notebook on Colab? it seems like its mission a bunch of things to get it to run on there

**Comment by BalintKomjati** [comment]
here you go :)
colab adoption: https://github.com/BalintKomjati/smooth-infinite-zoom

**Comment by kadirnar** [comment]
> colab adoption: https://github.com/BalintKomjati/smooth-infinite-zoom

Thank you for the codes.
HF Demo: https://huggingface.co/spaces/ArtGAN/stable-diffusion-2-infinite-zoom-out

**Comment by BalintKomjati** [comment]
> Thank you for the codes. HF Demo: https://huggingface.co/spaces/ArtGAN/stable-diffusion-2-infinite-zoom-out
cool app thanks! 
looks very smooth! :) does this have anything from my version?

**Comment by kadirnar** [comment]
Yes, I updated your code.  Thank you very much. This demo is really good.
